src/vs/editor/contrib/codeAction/browser/media/action.css

```diff
@@ -79,15 +79,22 @@
 }
 
 .codeActionMenuWidget .monaco-list .option-disabled,
-.codeActionMenuWidget .monaco-list .option-disabled .focused {
-	pointer-events: none;
+.codeActionMenuWidget .monaco-list .option-disabled:before,
+.codeActionMenuWidget .monaco-list .option-disabled .focused,
+.codeActionMenuWidget .monaco-list .option-disabled .focused:before {
+	cursor: default !important;
 	-webkit-touch-callout: none;
 	-webkit-user-select: none;
 	-khtml-user-select: none;
 	-moz-user-select: none;
 	-ms-user-select: none;
 	user-select: none;
+	background-color: var(--vscode-menu-background) !important;
 	color: var(--vscode-disabledForeground) !important;
+	outline: 0px solid !important; /* we still need to handle the empty tree or no focus item case */
```

Comment:
The growing number of importants seem fishy. Can you add a more specific css rule so that you don't need to use `!important`?

Reply:
I think I can remove these actually - we were having that weird issue where in the insiders build, the css was lower down so it was overwriting some of the rules, but was fixed earlier.